app/views/comments/_reply.html.erb

```diff
@@ -0,0 +1 @@
+

```

Comment:
If its not needed here then lets delete it

Reply:
I need it for reply_to_comments part.